skyportal/handlers/api/db_stats.py

```diff
@@ -84,4 +86,18 @@ def get(self):
         data["Newest candidate creation datetime"] = (
             cand.created_at if cand is not None else None
         )
+        data["Latest cron job run times & statuses"] = []
+        cron_job_scripts = DBSession().query(CronJobRun.script).distinct().all()
+        for script in cron_job_scripts:
+            cron_job_run = (
+                CronJobRun.query.filter(CronJobRun.script == script[0])
+                .order_by(CronJobRun.created_at.desc())
+                .first()
+            )
+            data["Latest cron job run times & statuses"].append(
```

Comment:
How does this filter out only the latest cron job results?

Reply:
see line 90 - we grab all the distinct "script" values (the cron job script filenames), then for each of those, grab the newest record

Reply:
:+1: